spec/models/validations/tenancy_validations_spec.rb

```diff
@@ -106,6 +106,25 @@
             expect(record.errors["tenancy"]).to be_empty
           end
         end
+
+        context "when referral is not internal transfer" do
+          it "adds an error" do
+            record.tenancy = "Assured"
+            record.referral = "Internal transfer"
+            tenancy_validator.validate_tenancy(record)
+            expect(record.errors["tenancy"])
+              .to include(match I18n.t("validations.tenancy.internal_referral"))
+          end
+        end
+
+        context "when referral is internal transfer" do
+          it "does not add an error" do
```

Comment:
The test description here is a little unclear - I had to look at your code under test to realise that the error should only apply if the referral is an internal transfer but the tenancy type is not given as secure (including flexible)

Reply:
`context "when type of tenancy is secure"` > `context "when referral is not internal transfer"` >  `it "adds an error"`
`context "when type of tenancy is secure"` > `context "when referral is internal transfer"` >  `it "does not add an error"`

Is that because the top most context is too high up?